src/native/libs/System.Security.Cryptography.Native/openssl.c

```diff
@@ -102,6 +104,8 @@ otherwise.
 */
 const ASN1_TIME* CryptoNative_GetX509NotBefore(X509* x509)
 {
+    // No error queue impact.
```

Comment:
I almost preemptively removed all of these comments, since they aren't as relevant with "keep using the last error in the queue".  Clearly, I didn't.

* Why remove: They were added to justify a lack of ERR_clear_error, when ERR_clear_error was necessary producing correct exceptions.
* Why keep: They represent the output of white-box investigation of the code behind the code.
* Why remove: They could be invalidated.
* Why keep: Uh... it made opening the PR easier.

I have to do something to deal with merge conflicts no matter what... so thought I'd open the floor for a keep or cut discussion.